release.sh

```diff
@@ -0,0 +1,91 @@
+set -e
+
+check_version () {
+	if ! [[ $1 =~ ^v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]
+	then
+	    echo -e "\nThat version does not match semver"
+	    exit
+	fi
+}
+
+get_branch () {
+	# Get the branch to release and abort if branch does not exist or is not checked out
+	echo "What branch do you want to release?"
+	read branch
+	git rev-parse --verify $branch
+	current_branch=`git rev-parse --abbrev-ref HEAD`
+
+	if ! [[ $current_branch = $branch ]]
+	then
+	    echo -e "\nCheckout branch you want to tag first. Aborting..."
+	    exit
+	fi
+}
+
+get_hash () {
+	current_hash=`git rev-parse HEAD`
+}
+
+get_new_version () {
+	# Get new version to release
+	echo -e "\nWhat version do you want to release?"
+	read new_version
+	check_version $new_version
+}
+
+get_prev_version () {
+	# Get previous version to generate release notes
+	echo -e "\nWhat is the previous version? (used to generate release notes)"
+	read prev_version
+	check_version $prev_version
+}
+
+print_release_notes () {
+	# Print release notes
+	echo -e "\nRelease notes: "
+	release_notes=`git --no-pager log $prev_version...$current_hash --no-merges --oneline`
+	echo -e "$release_notes"
+}
+
+confirm_tagging () {
+	# prompt to continue
+	read -p "Are you sure you want to update the version and tag? <y/N> " prompt
+	if ! [[ $prompt =~ [yY](es)* ]]
+	then
+		echo "Aborting..."
+		exit
+	fi
+}
+
+update_version () {
+	# update and commit local version file used by tracking telemetry
+	echo -e "\nWriting version file..."
+	echo $new_version > version
+	
+	echo -e "\nCommitting version file..."
+	git add version
+	git commit -m "[BUILD] Update version with latest release"
```

Comment:
I think we could just use `"[RELEASE] ${new_version}"` as the commit message here.